.dev/docker/doc/Dockerfile

```
@@ -1,4 +1,4 @@
-FROM pointcloudlibrary/env:20.04
+FROM pointcloudlibrary/env:21.04
```

```suggestion
FROM pointcloudlibrary/env:22.04
```
@SunBlack I updated the image on [dockerhub](https://hub.docker.com/r/pointcloudlibrary/doc/tags) to use Ubuntu 22.04 (with doxygen 1.9.1). I would propose:
1. You update the line here so that it matches the image on dockerhub
2. The doc-CI runs again, we check if there are any (new) warnings
3. Then this PR is probably ready for merge

IMO `pointcloudlibrary/env:latest` would be a bit ambiguous because we have Ubuntu and Windows images there. Long-term, we could try to have this docker depend on `ubuntu:latest` or `ubuntu:rolling` as you proposed earlier, but for now, I think it is fine to use the long-term-supported 22.04.
What do you think?

---

Couldn't we just use `doc:latest` now?

---

> Couldn't we just use `doc:latest` now?

This is the `doc` docker image, we can't base it on itself :slightly_smiling_face:

---

Ops you are right 😄 . So `ubuntu:latest` would be indeed the better solution than `env:latest`.

---

Yes, although if we base it on `ubuntu:latest`, we will have to install some things that are already included in `env:22.04` (e.g. cmake, flann, vtk, ...). On the plus side, we would have a smaller `doc` image since we don't need everything from `env:22.04`